lib/content-services/i18n/en.json

```diff
@@ -233,6 +233,24 @@
           "PROCESS_ACTION": "Start Process"
         }
       }
+    },
+    "FACET_FIELDS": {
+      "1_TYPE": "1:Type",
```

Comment:
Numbers were added just for demo shell, so that QA spot the wrong ordering/sorting. They should never be part of the content services i18n resources.

Reply:
Alternatively, you can move those strings to the demo shell resources

Reply:
ok, thanks Denys. I'll make the changes